### ISSUE TITLE
What version of pl are you using and what dataset are you using to train?

### ISSUE THREAD

**Issue by cooked-sashimi** [VCasecnikovs/Yet-Another-YOLOv4-Pytorch#10]
I'm trying to run your code, but it never seems to get into the `training_step` callback of the module and also I'm always getting `nan` for the loss. I'm using version `1.0.5` if that helps, also where have you downloaded the dataset for training?

**Comment by VCasecnikovs** [comment]
Hello, I am using pl 0.7.6. They have changed a lot the API of schedulers in the next versions. I also had this problem in one of my projects. For the dataset loader, You can use dataset.py class, it uses YOLO format or You can implement it by yourself.

**Comment by cooked-sashimi** [comment]
Thank you I have figured out what the problem was, in `ranger.py` the `closure` isn't being called and as a result the training step wasn't being called. After I uncommented it out everything worked! Thanks

**Comment by VCasecnikovs** [comment]
Was the problem on Your side. If on mine, could You please do pool request?

**Comment by cooked-sashimi** [comment]
Well in your `ranger.py` you comment out the `closure` [here](https://github.com/VCasecnikovs/Yet-Another-YOLOv4-Pytorch/blob/d010272d0483ca20d7f5c06bc2146d01f9e36164/ranger.py#L92) I just commented it out

**Comment by VCasecnikovs** [comment]
Thank you. 0b43789
